e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/PipelineRunFormAO.java

```diff
@@ -397,6 +398,19 @@ public int minNodeTypeRAM() {
                 .min().getAsInt();
     }
 
+    public PipelineRunFormAO doNotMountStoragesSelect (boolean isSelected) {
+        if ((!get(DO_NOT_MOUNT_STORAGES).has(cssClass("ant-checkbox-checked")) && isSelected) ||
+                (get(DO_NOT_MOUNT_STORAGES).has(cssClass("ant-checkbox-checked")) && !isSelected))    {
+            get(DO_NOT_MOUNT_STORAGES).click();
```

Comment:
It would be better to use click() of AccessObject class

Reply:
Done